clang-tools-extra/test/clang-doc/json/specialization-mangled-name.cpp

```diff
@@ -0,0 +1,15 @@
+// RUN: rm -rf %t && mkdir -p %t
+// RUN: clang-doc --output=%t --format=json --executor=standalone %s
+// RUN: FileCheck %s < %t/GlobalNamespace/_ZTT7MyClassIiE.json
```

Comment:
I guess this serves the same purpose, but it'd be nice to have an explicit check for the correct file name. Maybe just a call to `ls %t/GlobalNamespace/ | FileCheck %s --check-prefix=FILENAMES`? May not add much value though. @petrhosek WDYT?